modules/src/ics03_connection/query.rs

```diff
@@ -105,6 +119,24 @@ impl IdentifiedConnectionEnd {
     }
 }
 
+fn proto_unmarshal(bytes: Vec<u8>) -> Result<ConnectionEnd, error::Error> {
+    let buf = Bytes::from(bytes);
+    let decoded = proto_ibc_connection::ConnectionEnd::decode(buf);
+    match decoded {
+        Ok(conn) => {
+            let client_id = ClientId::from_str(&conn.client_id).unwrap();
```

Comment:
We should `.map_err` here (also for the other `unwrap()`s) and send the error up. For a command like this:
`cargo run --bin relayer -- -c ./relayer/relay/tests/config/fixtures/relayer_conf_example.toml query connection end chain_A connectionnotthere` we should get a friendlier output.